lib/NeuraDialect/Transforms/MapToAcceleratorPass.cpp

```diff
@@ -42,18 +41,56 @@ struct MapToAcceleratorPass
   Option<std::string> mappingStrategy{
       *this, "mapping-strategy",
       llvm::cl::desc("Mapping strategy to use for mapping operations to the "
-                     "accelerator. Options: greedy_mapping"),
-      llvm::cl::init("greedy_mapping")};
+                     "accelerator. Options: exhaustive, "
+                     "backtrack=max_loc,max_depth (default "
+                     "max_loc=5, max_depth=3), heuristic"),
+      llvm::cl::init("backtrack")};
 
   void runOnOperation() override {
     ModuleOp module = getOperation();
 
+    StringRef mappingStrategy_stringRef(mappingStrategy.getValue());
     // Creates a mapping strategy based on the provided option.
     std::unique_ptr<MappingStrategy> mapping_strategy;
-    if (mappingStrategy == "greedy_mapping") {
-      mapping_strategy = std::make_unique<GreedyMapping>();
-    } else if (mappingStrategy == "backtrack_mapping") {
-      mapping_strategy = std::make_unique<BacktrackMapping>();
+    if (mappingStrategy_stringRef == "heuristic") {
+      mapping_strategy = std::make_unique<BacktrackMapping>(INT_MAX, 1);
+    } else if (mappingStrategy_stringRef == "exhaustive") {
+      mapping_strategy = std::make_unique<BacktrackMapping>(INT_MAX, INT_MAX);
+    } else if (mappingStrategy_stringRef == "backtrack") {
```

Comment:
Can we rename `heuristic` to `greedy`, `backtrack` to `heuristic`? (backtrack is super standard, no need to expose to users, WDYT?)